contents/docs/feature-flags/bootstrapping.mdx

```diff
@@ -74,9 +74,9 @@ The `bootstrap` object has two optional arguments: `distinctID` and `isIdentifie
 
 - `distinctID` enables you to set distinct ID of the user during PostHog's initialization. This is useful when you want to ensure the distinct ID on your frontend is the same as the distinct ID that you called `getAllFlags()` with on your server. It is **strongly recommended** to include it.
 
-> **Note:** The only case where it doesn't make sense to include the distinctID is when you're bootstrapping identified users only, and you call identify for these users right on load. Here, if you pass in the distinctID to bootstrap, identify will not connect your existing anonymous ID, since it's replaced by the bootstrapped ID.
+> **Note:** The only case where it doesn't make sense to include `distinctID` is when your code only runs for [identified users](/docs/product-analytics/identify) **and** you call `identify` immediately on load. 
```

Comment:
Thanks Neil! I get it now. Is the below better?

```suggestion
> **Note:** The only case where you don't want to include `distinctID` is when your user has an existing PostHog session and you have not yet called [`identify()`](/docs/product-analytics/identify) at any point in time in order to link the anonymous session ID with their identified ID. In this case, bootstrapping `distinctID` will cause PostHog to drop the anonymous session ID and the two sessions will not be linked.
```

Reply:
excellent, yeah much better! 🙌

Reply:
Hmm,  one bit that's missing is that in this case we definitely want them to call identify after bootstrapping, so the right ID is used, as otherwise there can be a mismatch between the user ID, and the flags being bootstrapped